src/utilities/utilities.jl

```diff
@@ -11,6 +11,9 @@ using .Remotes: Remote, repourl, repofile
 using .Remotes: RepoHost, RepoGithub, RepoBitbucket, RepoGitlab, RepoAzureDevOps,
     RepoUnknown, format_commit, format_line, repo_host_from_url, LineRangeFormatting
 
+const original_pwd = Ref{String}()  # for printing relative paths in error messages
```

Comment:
I'd like this type of state to be stored in the `Document` object, and to avoid global state. I think that should be feasible here?

Reply:
Oh.. we don't pass the context down to `locrepr` now. Would it feasible to change all the call sites...? Otherwise, ScopedValues could be an option..

Reply:
And `ScopedValues` require Julia >= 1.11 ...

Reply:
Even `ScopedValues.jl` require Julia >= 1.8

Reply:
I guess `task_local_storage` could be another option. But.. actually.. Documenter is not async/multi-thread safe anyway (like, the `cd`s that we do, for example). So let's just stick with the global variable for now -- we have some precedent, and it is actually the simplest one to find (since it's quite explicit if you have some dynamic global variables around).